pkg/proto/datadog/api/v1/api.proto

```diff
@@ -178,6 +179,14 @@ service AgentSecure {
             body: "*"
         };
     };
+
+    // Get the host tags
+    rpc GetHostTags(datadog.model.v1.HostTagRequest) returns (datadog.model.v1.HostTagReply) {
+        option (google.api.http) = {
+            get: "/v1/grpc/hosttags"
```

Comment:
Seems like it should be `host_tags` to match the format of other RPC calls.

Reply:
Updated 👍